app/components/pages/Staff/EditorialPage.js

```diff
@@ -13,19 +13,19 @@ const EditorialPage = () => (
       <StaffCard
         jobTitle="Editorial Manager"
         name="Wei Mun Chan"
-        photoURL="https://submit.elifesciences.org/images/WeiMun.png"
+        photoURL="https://prod--journal-cms.elifesciences.org/sites/default/files/iiif/person/2017-07/weimun.jpg"
         telephone="+44 [0]1223 855379"
       />
       <StaffCard
         jobTitle="Journal Development Editor"
         name="Maria Guerreiro"
-        photoURL="https://submit.elifesciences.org/images/Maria.png"
+        photoURL="https://prod--journal-cms.elifesciences.org/sites/default/files/iiif/person/2017-07/maria.jpg"
         telephone="+44 [0]1223 855376"
       />
       <StaffCard
         jobTitle="Senior Editorial Assistant"
         name="Susanna Richmond"
-        photoURL="https://submit.elifesciences.org/images/Susanna.png"
+        photoURL="https://prod--journal-cms.elifesciences.org/sites/default/files/iiif/person/2017-07/susanna.jpg"
```

Comment:
How can we record that these files are being used in xPub as well as Journal? I can imagine someone renaming, updating or moving these in Journal without realising that it breaks the xPub pages.

Reply:
Very good point! ... @giorgiosironi  @thewilkybarkid ???

Reply:
Hard-coding any of these details looks like a bad idea, but you can use the `/people` API to fetch the IIIF path (so you can make it whatever size you need etc).

Reply:
<img src="https://thumbs.dreamstime.com/z/businessman-ceo-smart-acting-isolated-white-background-51951316.jpg" width=200 title="Feeling like an executive" />

Reply:
And, `iiif.elifesciences.org` is not just a dynamic resizing service but also a CDN with cache, edge nodes located across the world, etc so it's the best way to access an image